src/api/app/models/user.rb

```diff
@@ -10,6 +10,9 @@ class User < ApplicationRecord
   NOBODY_LOGIN = '_nobody_'.freeze
   MAX_BIOGRAPHY_LENGTH_ALLOWED = 250
 
+  COLOR_THEMES = ['light', 'dark'].freeze
```

Comment:
I'll replace "light" with "system" instead because we will only have two options and we won't craft a "light" theme right away.

Reply:
So, in the case of "system" you want to use the one coming from the OS setting, right?  We could check with `@media (prefers-color-scheme: dark)` to add the overrides. But in the code, I used `@include color-mode(dark)` to define the overrides which are based on the `data-bs-theme` attribute from the HTML. 

I'm not sure how to use one or the other depending on the value of the column and point to the same dark theme without repeating all the overrides inside the `prefers-color-schema: dark` section. We need to specify `$color-mode-type`, which controls the if you are based on the `media-query` or the bootstrap color scheme system... and you can only choose one or the other.

Reply:
Ok, I added a small javascript snippet to handle the values coming from DB. If the user selects `system` in the profile, it uses the `prefers-color-scheme` from the browser, otherwise, it goes straight to that value set in the profile.